### PR TITLE
Add info on sending quit requests properly in Handling quit requests

### PR DESCRIPTION
**Pull request by insomniacUNDERSCORElemon**
Particularly needed as most info online suggests the force-quit way.

I am a lurker, feel free to clean this up if needed (I just figured filing an issue was 90% of the way there already).

### CODE REVIEW

File: tutorials/inputs/handling_quit_requests.rst [hunk 1]
```diff
@@ -59,3 +59,19 @@ behavior to quit when quit is requested, this can be changed:
...
+Insead you should be sending a quit request:
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    get_tree().notification(MainLoop.NOTIFICATION_WM_QUIT_REQUEST)
+
+ .. code-tab:: csharp
+
+    GetTree().Notification(MainLoop.NotificationWmQuitRequest)
```
Comment (Calinou) [inline tutorials/inputs/handling_quit_requests.rst:77]:
```suggestion
While forcing the application to close can be done by calling :ref:`SceneTree.quit <class_SceneTree_method_quit>, doing so will not send the quit *notification*. This means the function described above won't be called. Quitting by calling :ref:`SceneTree.quit <class_SceneTree_method_quit> will not allow custom actions to complete (such as saving, confirming the quit, or debugging), even if you try to delay the line that forces the quit. 

Instead, you should send a quit request:

.. tabs::
 .. code-tab:: gdscript GDScript

    get_tree().notification(MainLoop.NOTIFICATION_WM_QUIT_REQUEST)

 .. code-tab:: csharp

    GetTree().Notification(MainLoop.NotificationWmQuitRequest)
```

File: tutorials/inputs/handling_quit_requests.rst [hunk 2]
```diff
@@ -59,3 +59,24 @@ behavior to quit when quit is requested, this can be changed:
  .. code-tab:: csharp
 
     GetTree().SetAutoAcceptQuit(false);
+
+Sending your own quit notification
+----------------------------------
+
+While forcing the application to close can be done by calling :ref:`SceneTree.quit <class_SceneTree_method_quit>,
```
Comment (Calinou) [inline tutorials/inputs/handling_quit_requests.rst:66]:
```suggestion
While forcing the application to close can be done by calling :ref:`SceneTree.quit <class_SceneTree_method_quit>`,
```

File: tutorials/inputs/handling_quit_requests.rst [hunk 3]
```diff
@@ -59,3 +59,24 @@ behavior to quit when quit is requested, this can be changed:
...
+doing so will not send the quit *notification*. This means the function
+described above won't be called. Quitting by calling
+:ref:`SceneTree.quit <class_SceneTree_method_quit> will not allow custom actions
```
Comment (Calinou) [inline tutorials/inputs/handling_quit_requests.rst:69]:
```suggestion
:ref:`SceneTree.quit <class_SceneTree_method_quit>` will not allow custom actions
```

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Comment by insomniacUNDERSCORElemon** [comment]
Happy to help when/if I can. Although technically it hasn't been merged yet (at time of writing this).

**Comment by Calinou** [comment]
Cherry-picked to the `3.3`, `3.2` and `stable` branches.

**Pull request merged**